### PR TITLE
Relicense from AGPL to MIT

### PR DESCRIPTION
**Pull request by logan**
When we open sourced heim, we chose to license the backend code under AGPL, and the client code under MIT. The rationale was that the AGPL would impose some pressure to keep us honest in certain ways (as well as anyone else operating a community with heim).

Now that the business is more or less dissolved, I don't believe AGPL provides strategic value to the project (if it ever did). It also complicates my relationship with my employer if I want to continue contributing to this project. I live in California, where it works out best to share the IP I develop for heim with my employer. I can do that with the MIT license, but not with AGPL.

I need CylonicRaider to expressly relicense their contributions under the MIT license. I figure we may as well do this via code review. If you approve this code review, you are licensing your past contributions to heim under the terms of the MIT license.

I'd also like the approval of everyone else I've asked to review this change. Technically Euphoria Ops owns your past contributions but I'd like everyone who has contributed under the terms of the AGPL to be on board with relicensing. Please reply if you have any objections, questions, or concerns, and approve if you agree with the change.

### CODE REVIEW

**Comment by CylonicRaider** [comment]
I feel honored. :bowing_man:
Scanning the contributor list reveals... uh... that indeed everyone who probably has contributed to the backend is already covered.

**Comment by logan** [comment]
Yes, this was my finding:

```
$ git shortlog -s -n aws backend cluster heimctl
   409  Logan Hanks
    26  Max Goodman
    21  Cameron Palone
     3  CylonicRaider
     1  Cam Palone
```

File: LICENSE.md [hunk 1]
```diff
@@ -1,31 +1,20 @@
...
-
 ## art, doc, and client/res
 
 Files under the `art`, `doc`, and `client/res` directories are offered under
 the terms of the [CC BY 4.0](http://creativecommons.org/licenses/by/4.0/)
 license.  You'll find a copy in the `LICENSE` file in each of these
 directories.
 
-## client and all else
+## aws, backend, client, cluster, heimctl, and all else
```
Comment (CylonicRaider) [inline LICENSE.md]: Given that there is no differentiation from the ‘backend’ now, and that is redundant with ‘all else’, one might consider dropping the explicit list.
The directories *not* named yet (and not starting with underscores) are `ci`, `containers`, `heimlich`, `proto`, and `templates` (the last three including source files that *could* have to do with the backend as well :thinking:).

**Comment by CylonicRaider** [comment]
The additional three directories I've found that contain Go files add no new contributors (leaving @ckingdev's names aside):
```
$ git shortlog -s -n heimlich proto templates
   188  Logan Hanks
    10  Max Goodman
     5  Cameron Palone
     4  M. Cameron Palone
```

**Comment by CylonicRaider** [comment]
A point that perhaps deserves some attention is updating the top-level README file, which has a section on licensing as well.

**Pull request merged**